tests/components/command_line/test_switch.py

```diff
@@ -595,9 +595,9 @@ async def test_templating(hass: HomeAssistant) -> None:
         entity_state = hass.states.get("switch.test")
         entity_state2 = hass.states.get("switch.test2")
         assert entity_state.state == STATE_ON
-        assert entity_state.attributes.get("icon") == "mdi:on"
+        assert entity_state.attributes.get("icon") == "mdi:off"
         assert entity_state2.state == STATE_ON
-        assert entity_state2.attributes.get("icon") == "mdi:on"
+        assert entity_state2.attributes.get("icon") == "mdi:off"
```

Comment:
These tests look like they end up incorrect now? 

State On but icon off?

Reply:
It is correct but I agree it looks strange with these examples. 
I couldn't figure better examples yesterday but I'll take a second look later and see if we can do something that makes sense.

Reply:
It's not pretty but I guess it makes more sense now with the tests...